docker/docker-compose.yml

```diff
@@ -610,3 +614,54 @@ services:
     command: "./wait-for-it.sh mariadb:3306 --timeout=720 --strict -- ./start.sh"
     profiles:
       - bitcoin-mempool
+
+  prometheus:
+    image: prom/prometheus:v2.45.2
```

Comment:
nit: v3.0.1

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1105/commits/28a04360ff6f6208c8732ce53802bca21b45af7e.